ProjFS.Mac/PrjFSKext/VirtualizationRoots.cpp

```diff
@@ -664,14 +665,32 @@ bool VirtualizationRoots_ProcessMayAccessOfflineRoots(pid_t pid)
     
     RWLock_AcquireShared(s_virtualizationRootsLock);
     {
-        for (uint32_t i = 0; i < s_offlineIOPIDCount; ++i)
+        while (true)
```

Comment:
Can we please add some comments here.  While true's in locks can be scary :-).

Reply:
I've added some comments to this function - please let me know if there's any further detail I should cover in comments.